client/my-sites/plan-features-2023-grid/_media-queries.scss

```diff
@@ -10,9 +10,10 @@ $plans-2023-medium-breakpoint: 1200px;
 $minWidthToGridWidthMap: (
 	780px: "668px",
 	1024px: "860px",
-	1200px: "1134px",
-	1440px: "1320px",
-	1600px: "1480px",
+	1200px: "100%",
+	// 1200px: "1134px",
+	// 1440px: "1320px",
+	// 1600px: "1480px",
```

Comment:
Let's remove these commented lines unless there is a special reason :)

Reply:
I second this